crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -53,8 +53,10 @@ const MAX_BLOB_LIMIT: usize = 128;
 /// ## Implementers
 ///
 /// Implementing support for an engine API jsonrpsee RPC handler is done by defining the engine API
-/// server trait and implementing it on a type that can wrap this [`EngineApi`] type.
-/// See also [`EngineApiServer`] implementation for this type which is the L1 implementation.
+/// server trait and implementing it on a type that can either wrap this [`EngineApi`] type or
+/// provide an [`EngineTypes`] implementation if it mirrors ethereum's versioned engine API
```

Comment:
I like the updated and extended info, but it makes me wonder how do you "provide an [`EngineTypes`] implementation if it mirrors ethereum's versioned engine API endpoints?" I feel like it may not be visible from here.

Reply:
good point, adding some more context here